cpp_json_kw.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef CPP_JSON_KW__HPP
+#define CPP_JSON_KW__HPP
+
+#include <optional>
+#include <string>
+#include "json.hpp"
+#include <stdexcept>
+
+using nlohmann::json;
+
+template <typename T>
+struct mapped_json_value {bool success ; std::optional<T> value;};
+
```

Comment:
Comment about intent of `kws_get`

Reply:
Also, is the boolean return value needed ? use has_value or cmpare against std::nullopt

Reply:
make an issue around it, don't fix maybe until point-2 release

Reply:
Now an issue: https://github.com/irods/irods_capability_indexing/issues/71